resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -746,6 +746,17 @@ sqs.send.message.deduplication_id=Message Deduplication ID:
 sqs.send.message.group_id=Message Group ID:
 sqs.send.message.send.button=Send
 sqs.send.message.success=Sent message ID: {0}
+sqs.service_name=Amazon SQS
 sqs.standard.queue.tooltip=Supports at-least-once delivery and message ordering is not preserved.
+sqs.subscribe.sns=Subscribe to SNS topic
+sqs.subscribe.sns.failed=Failed to subscribe {0} to {0}
+sqs.subscribe.sns.in_progress=Subscribing...
+sqs.subscribe.sns.select=Select SNS topic
+sqs.subscribe.sns.select.tooltip=When a message is published to a topic, Amazon SNS sends the message to every subscribed queue.
```

Comment:
I think shorten to something like `messages from this topic are sent to query {0}`?
or it's not needed at all.